scripts/pi-hole/php/groups.php

```diff
@@ -640,15 +640,9 @@ function verify_ID_array($arr)
                 }
             }
 
-            if(isset($_POST['type']) && strlen($_POST['type']) === 2 && $_POST['type'][1] === 'W')
-            {
-                // Apply wildcard-style formatting
-                $domain = "(\\.|^)".str_replace(".","\\.",$domain)."$";
-            }
-
-            if($type === ListType::whitelist || $type === ListType::blacklist)
+            if( ! ($_POST['type'] == '2' || $_POST['type'] == '3'))
```

Comment:
```suggestion
            if( $_POST['type'] != '2' && $_POST['type'] != '3')
```

Just think this is easier to follow...